ha/core/config/config_manager.py

```diff
@@ -126,3 +129,40 @@ def get_hw_env() -> str:
         Get if system is running on VM or actual h/w.
         """
         return Conf.get(const.HA_GLOBAL_INDEX, f"CLUSTER_MANAGER{_DELIM}env")
+
+    @staticmethod
+    def get_node_id(node_name: str) -> str:
+        """
+        Get node_id from node_name
+        Args:
+            node_name (str): Node name from cluster nodes.
+        Returns: str
+            node_id (str): Node ID from cluster nodes.
+        """
+        confstore = ConfigManager.get_confstore()
+        key_val = confstore.get(f"{const.PVTFQDN_TO_NODEID_KEY}/{node_name}")
+        _, node_id = key_val.popitem()
+        return node_id
+
+    @staticmethod
+    def get_online_nodes():
+        """
```

Comment:
we already have a routine _get_offline_nodes in node_stop branch in the ha/core/controllers/pcs/node_controller.py file.
Can we have both functions in same location ?

Reply:
@ArchanaLimaye  this code is for ha setup so I think its good to have this in config manager itself instead of keeping it in core location, later we can move get_offline_nodes here in config manager only.

Reply:
@indrajitzagade we can take it in code refactor, If we do this change now we need to test the alert monitor functionality again.

Reply:
@shindeam Please raise a refactor ticket.

Reply:
@indrajitzagade created ticket for code refactor : https://jts.seagate.com/browse/EOS-24729

Reply:
@shindeam  please mention the ID in the file

Reply:
Done